packages/connectivity/src/scp-cf/destination/destination-cache.spec.ts

```diff
@@ -343,7 +350,6 @@ describe('destination cache', () => {
       );
       const actual = await getDestination({
         destinationName: destName,
-        useCache: true,
```

Comment:
[req] There is also one left in line 325

Reply:
[req] Can you check the test `disables the cache by default`? This should not work anymore

Reply:
The test was set up with the wrong values to cache, hence was working without failing. 
I have fixed this now and added tests for both caching by default and non-caching case.